src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const query = document.querySelectorAll('tr');
```

Comment:
Как-то много кода получилось, тебе просто надо пройтись циклом по массиву этих строк таблицы, и для каждой из них сделать
```javascript
element.children[3].after(element.children[1].cloneNode(true));
```
Кстати почему переменная называется query?


```suggestion
const rows = document.querySelectorAll('tr');
```